x-pack/test_serverless/api_integration/test_suites/common/platform_security/role_mappings.ts

```diff
@@ -45,7 +45,7 @@ export default function ({ getService }: FtrProviderContext) {
 
         it('role mapping feature check', async () => {
```

Comment:
Nit: rename and move out of role_mappings folder
```suggestion
        it('security feature check', async () => {
```

Reply:
on this one, if decide to enable it for serverless too, we might not need this test at all.